contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -1570,6 +1570,9 @@ begin
   if pattern is null or expression is null then
     return null;
   end if;
+  if sys.is_collated_ai(expression) then
+    return sys.patindex_ai_collations(pattern, expression);
```

Comment:
If PATINDEX is not even parallel safe, why define patindex_ai_collations parallel safe ?

Reply:
remove parallel safe tag from declaration